Build/GameSettings.cc

```diff
@@ -186,7 +186,7 @@ void InitGameOptions()
 	gGameOptions.fSciFi						 = TRUE;
 	gGameOptions.ubDifficultyLevel = DIF_LEVEL_EASY;
 	//gGameOptions.fTurnTimeLimit		 = FALSE;
-	gGameOptions.fIronManMode			 = FALSE;
+	gGameOptions.ubGameSaveMode			 = DIF_CAN_SAVE;
```

Comment:
Yes, I'm all for that, since the code reads odd now. However it doesn't need to be taken care of in this PR. I'll open a new bug as a reminder.